templates/corporate/billing.html

```diff
@@ -198,27 +198,9 @@ <h3>
                 To make changes to your plan or view your billing history <a href="mailto:support@zulip.com">contact Zulip support</a>.
             </p>
         </div>
-        {% elif admin_access and not has_active_plan %}
-        <!-- For sponsored or sponsorship pending organizations we already redirect to /sponsorship page. -->
-        <div class="tab-content">
-            <center>
-                <p>
-                    <h2>Your organization is on the <b>Zulip Free</b> plan.</h2>
-                    <a href="/upgrade/">
-                        <button type="submit" class="button green">Upgrade to Standard</button>
-                    </a>
-                    <a href="/plans/">
-                        <button type="submit">View all plans</button>
-                    </a>
-                </p>
-            </center>
-        </div>
-        <div class="support-link">
-            <p>
-                <a href="mailto:support@zulip.com">Contact Zulip support</a> for billing history.
-            </p>
-        </div>
         {% else %}
+        <!-- For sponsored or sponsorship pending organizations we redirect to /sponsorship page. -->
+        <!-- For organizations on free plan we redirect to /plans page. -->
```

Comment:
Just a note that while this is correct for our current implementation, when we add a "billing history" feature, we may end up needing to undo this, so that organizations that used to be customers can see their past bills. Not a thing we will do in 2023 though.